qcodes/instrument_drivers/ZI/ZIUHFLI.py

```diff
@@ -952,6 +939,13 @@ def __init__(self, name: str, device_ID: str, **kwargs) -> None:
                                                 sigout-1, 1, 'range'),
                                 vals=vals.Enum(0.075, 0.15, 0.75, 1.5))
 
+            self.add_parameter('signal_output{}_ampdef'.format(sigout),
+                               get_cmd=None, set_cmd=None,
```

Comment:
Perhaps not very much related to this PR, but still. i'm not sure i understand how this parameter works. I see that it's used only in the private "_setter" function, but not used in the private "_getter" function. So, my questions is how does the instrument know that it has to accept or return values in Vpk/Vrms/dBm? Have i missed something? Or is this parameter just pure convenience for the users of the driver?

Reply:
I moved the parameter around and got the blame. I did not add it or change it. But that said, I don't really understand the value of this parameter. The device has its amplitude in V, i.e it sets the peak amplitude of the signal output and scales the signal accordingly. I only see it in the setter, so if you set it to dBm, and then set the amplitude with `signal_output{}_amplitude{}` then I guess you will not get the same value back when retrieving the amplitude. But maybe someone else has a better understanding of this parameter.

Reply:
>  and got the blame

no blame :) I was seeking clarification for the sake of better understanding the driver and the change in this PR, nothing more)

Alright, then let's keep everything as is. I guess the existence of this parameter does not affect, say, your work, right?

Reply:
No we do not use the parameter. When you view code on github, there is a blame button that you can click to see who did what. That´s what I ment with getting the blame. But I moved that parameter back, and some of the blame with it :)

Reply:
ah, ok, i see))